cmd/kubeadm/app/phases/upgrade/postupgrade.go

```diff
@@ -103,22 +114,41 @@ func PerformPostUpgradeTasks(client clientset.Interface, cfg *kubeadmapi.MasterC
 	return errors.NewAggregate(errs)
 }
 
-func coreDNSDeployment(cfg *kubeadmapi.MasterConfiguration, client clientset.Interface) error {
+func upgradeToCoreDNSIfPossible(cfg *kubeadmapi.MasterConfiguration, client clientset.Interface) error {
```

Comment:
Personally I feel like a bit confused by the func name :)

Reply:
fixed